src/ray/util/process.h

```diff
@@ -21,6 +21,7 @@
 #endif
 
 #include <functional>
+#include <map>
```

Comment:
After the `#include`s, add this:
```suggestion
#include <map>

#ifdef _WIN32
struct EnvironmentVariableComparator {
  bool operator()(char a, char b) const {
    // TODO(mehrdadn): When Process adds Unicode support, this should be updated.
    return std::less<char>()(tolower(a), tolower(b));
  }
  bool operator()(const std::string &a, const std::string &b) const {
    return std::lexicographical_compare(a, b, *this); 
  }
};
#else
typedef std::less<std::string> EnvironmentVariableComparator;
#endif
typedef std::map<std::string, std::string, EnvironmentVariableComparator> ProcessEnvironmentTable;
```